### ISSUE TITLE
Question about button cases

### ISSUE THREAD

**Issue by karltinsly** [climent/aluminum_pole_320#1]
I see that there are four possible button events, but it looks like only two are used in the main loop. Am I right that the four events are single press, double press, long press, and very long press? Do all four work, and can I add the other two events to the main loop? Thanks for sharing your code!

**Comment by climent** [comment]
Yes, those 2 other button operations are long press and very long press (3sec, 5sec) and both work. You can add triggers on those operations.
